info.yml

```diff
@@ -206,6 +206,14 @@ configuration:
         description: Controls whether new tasks can be created from the app.
         default_value: True
 
+    file_browser_tabs:
+        type: list
+        description: "Controls what tabs are visible in the main browser"
+        allows_empty: True
```

Comment:
Please set to `False`, otherwise it allows someone to configure the app without any tabs, which doesn't work.